toml.md

```diff
@@ -104,27 +104,39 @@ Keys
 
 A key may be either bare, quoted, or dotted.
 
-**Bare keys** may only contain ASCII letters, ASCII digits, underscores, and
-dashes (`A-Za-z0-9_-`). Note that bare keys are allowed to be composed of only
-ASCII digits, e.g. `1234`, but are always interpreted as strings.
+**Bare keys** may contain any letter-like or number-like Unicode character from any Unicode script,
+as well as ASCII digits, dashes and underscores. Punctuation, spaces, arrows, box drawing 
+and private use characters are not allowed.  Note that bare keys are allowed to be
+composed of only ASCII digits, e.g. `1234`, but are always interpreted as strings.
+
+* From the ASCII characters, only `A-Z`, `a-z`, `0-9`, `_` and `-` are allowed.
+* From the rest of the ISO-8859-1 characters, allowed are U+B1 to U+B3, U+B9, U+BC to U+BE, U+C0 to U+D6, U+D8 to U+F6 and U+F8 to U+FF
+* All of U+0100-1FFF are allowed, except `;` (U+037E).
+* Characters U+200C, U+200D, U+203F, U+2040, U+2070 to U+218F, U+2460 to U+24FF, and U+2C00 to U+2FEF are allowed.
+* All characters from U+3001 and higher, except surrogates (U+D800 to U+D999), Private Use (U+E000 to U+F8FF, U+F0000 to U+100000) and process-internal use (U+FDD0 to U+FDEF).
```

Comment:
Sorry for nitpicking, but this list and the paragraph before it should be reflowed to correspond to the 80 character line length used in the TOML spec.

Reply:
I wasn’t aware, will fix, tx

Reply:
Done